File: python/7 kyu/198.py
# https://www.codewars.com/kata/59dd2c38f703c4ae5e000014/train/python

# Passed

def solve(s:str) -> int:
    nums = []
    
    num = str()
    for idx, char in enumerate(s):
        if char.isdigit():
            num += char
        
        if num and (not char.isdigit() or idx == len(s) - 1):
            num = int(num)
            nums.append(num)
            num = str()
            
    largest_num = max(nums)
    return largest_num

output = solve('gh12cdy695m1')
print(output)

File: nodejs/7 kyu/198.js
// https://www.codewars.com/kata/5a1c28f9c9fc0ef2e900013b/train/javascript

// Passed

function pyramid(n) {
  let layers = [];
  for (let layerNumber = 0; layerNumber < n; layerNumber++) {
    let layer = " ".repeat(n - layerNumber - 1);
    if (layerNumber !== n - 1) {
      layer += "/" + " ".repeat(layerNumber * 2) + "\\";
    } else {
      layer += "/" + "_".repeat(layerNumber * 2) + "\\";
    }
    layers.push(layer);
  }

  let asciiPyramid = layers.join("\n") + "\n";
  return asciiPyramid;
}

const output = pyramid(10);
console.log(output);